pkgs/applications/misc/notable/default.nix

```diff
@@ -34,6 +34,9 @@ appimageTools.wrapType2 rec {
       $out/share/icons/hicolor/1024x1024/apps/notable.png
     substituteInPlace $out/share/applications/notable.desktop \
       --replace 'Exec=AppRun' 'Exec=${pname}'
+    source "${makeWrapper}/nix-support/setup-hook"
```

Comment:
```suggestion
```
add it to nativeBuildInputs

Reply:
`appimageTools.wrapType2` doesn't take a `nativeBuildInputs` argument

Reply:
Then we should add that if it is required to make this a little bit nicer

Reply:
That is definitely out of scope for this PR. Otherwise: I agree.

Reply:
Just had another look: The script gets passed down to `runCommandLocal` via `buildFHSEnv`, so `nativeBuildInputs` don't fit into this call stack.